bin/odilo_monitor_full

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why did you remove this script, is it not needed? Also, just a heads up that it will need to be removed from the crontab in the docker repo too.